tests/functional/ext/comparison_placement/misplaced_comparison_constant.py

```diff
@@ -1,6 +1,6 @@
 """Check that the constants are on the right side of the comparisons"""
```

Comment:
```suggestion
"""Check that the constants are on the right side of the comparisons"""

```

Reply:
It's a clever way to not have to update the functional output and reduce conflict, I'll leave it like this.